src/scripts/sims/sim_processors.ts

```diff
@@ -266,8 +267,13 @@ export class CycleProcessor {
     private readonly manuallyActivatedBuffs: readonly Buff[];
     combatStarted: boolean = false;
     readonly useAutos: boolean;
+    private readonly cdTracker: CooldownTracker;
```

Comment:
```suggestion
    readonly cdTracker: CooldownTracker;
```
If this is `private`, it can't be queried from a sim to check the cooldown status of an ability before trying to use it.